src/features/SelectedPost/SelectedPost.ts

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-param-reassign, @typescript-eslint/no-unused-vars */
```

Comment:
change file (and folder, but actually you may get rid of folder at all) name to selectedPost.ts (camelCase)